docs/3-managed-service.md

```diff
@@ -168,50 +133,34 @@ Changing your username is easy but can lead to confusion for end-users, especial
 
 Like when changing your display name, when you change your username, you'll lose verification and need to go through the process again.
 
-### Email & PGP
-
-<img src="img/settings.email.png">
-
-#### Email Delivery
-
-Users can have messages delivered to an email address and with the SMTP service of their choosing.
-
-#### PGP
-
-Users can opt to have their messages encrypted so they're only readable by them. This is a highly encouraged option, especially for journalists.
+### Email & Encryption
 
-### Advanced
-
-#### Delete Account
-
-Easily and permanently delete your account whenever you want.
+Before users can enable message forwading, a PGP key must be uploaded.
 
-<img src="img/settings.advanced.png">
+![Settings - Email](https://github.com/user-attachments/assets/4a1ec530-cf19-432e-81ee-cbbf53da1261)
 
-## Admin View
-
-Admins can control certain aspects of user settings without having to log in to the server or manually modifying the database. This is where users are verified, where paid features can be manually enabled, or where admins can be created.
+#### Email Delivery
 
-A highlight panel displays the total number of users and how many have enabled 2fa or added a PGP key. This view helps with internal decision-making, like education campaigns for security best practices.
+We use Riseup.net for our email forwarding provider. All you need to do is add an email address and you'll get new secure messages delivered directly to your inbox.
```

Comment:
It might be worth being explicit here as Hush Line also has its own "inbox"
```suggestion
We use Riseup.net for our email forwarding provider. All you need to do is add an email address and you'll get new secure messages delivered directly to your email inbox.
```